clients/admin-ui/src/features/common/fides-types.ts

```diff
@@ -1,20 +0,0 @@
-/**
- * Common Fides types adapted from https://github.com/ethyca/fideslang/blob/main/src/fideslang/models.py
- */
-
-export type FidesKey = string;
```

Comment:
There are a few situations where aliasing a primitive type is helpful for documentation purposes, but I'd agree this isn't one of them.